### DISCUSSION TITLE
implementation in other boards

### DISCUSSION THREAD

**Discussion by kleffa** [darthcloud/BlueRetro#316] [Ideas]
Hello, I'm new in this so I have to ask ESP32-DEVKITC-32E is the only esp32 board that can be used for blueretro? There is a way to use other model or it is a hardware limitation?

Thanks for your proyect, I'm looking to implement this on a family Wii just soldering to Wii boards pins

**Comment by darthcloud** [comment]
You can use any ESP32 board but the only one I test and can help with is the ESP32-DEVKITC-32E.

People often got issues with board using "clone" module.

**Reply by kleffa** [reply]
im trying it now in a lolin 32 lite, why are too many bins on relases? there is one that only works on gamecube?

**Reply by kleffa** [reply]
@darthcloud 
![IMG-20220803-WA0003](https://user-images.githubusercontent.com/26838533/183568363-542b6ccd-31e1-470a-b402-5bedd5afaab7.jpeg)
It works with the GC bin, since my Wii console has no GC ports I solder it to board data pins, and 3.3v.... I used a lolin32 lite since is smaller than devkit, but doesn't have all pins to use in other consoles. I guess a bare ESP32 wroom or a lolin like this can be powered by a Wiimote/mini s/nes

**Comment by persechini** [comment]
I was wondering about something similar. I've just come across the lolin s3 mini, seems to me a great fit for blueretro but the available pins are different, a couple of GPIO pins seem not to be available (for my uses, SNES/PSX/N64). Is it possible to compile the firmware using a different pinout to make use of this particular board?
